spec/acceptance/ios_aaa_accounting_spec.rb

```diff
@@ -1,6 +1,43 @@
 require 'spec_helper_acceptance'
 
 describe 'ios_aaa_accounting' do
+  before(:all) do
```

Comment:
Is this a rebase issue? Please fix up the commits so it is clear what this belongs to.